src/main/org/jamocha/parser/clips/Token.java

```diff
@@ -97,16 +97,12 @@ public Token(int kind, String image)
   /**
    * Returns the image.
    */
+  @Override
   public String toString()
   {
     return image;
   }
 
-  public void clear() {

```

Comment:
clear is important for performance in the older versions of JavaCC. With version 2 and 3, JavaCC holds on to the entire char stream until parsing is done. I don't know if the latest version fixed that old bug, but it would be good to test the change with a larger clips file that is bigger than 1MB. You'll see clips.jj calls clear() in several places to make sure memory usage is minimal. Back when I benchmarked this in 2004, small files less than 50K had no difference. It was when I tried to load data that I saw memory spike and parsing slowed down 5-10x.